src/main/java/roomescape/auth/AuthController.java

```diff
@@ -45,8 +56,9 @@ public ResponseEntity<Void> login(@RequestBody @Valid LoginRequest loginRequest)
     public ResponseEntity<LoginCheckResponse> loginCheck(HttpServletRequest request) {
         Cookie[] cookies = request.getCookies();
         String accessToken = cookieResolver.getToken(cookies);
+        String name = jwtResolver.getName(accessToken);
 
-        LoginCheckResponse result = authService.checkAccessToken(accessToken);
+        LoginCheckResponse result = new LoginCheckResponse(name);
         return ResponseEntity.ok()
```

Comment:
[제안 🔈 ]

다음과 같이 ResponseEntity.ok(body)로 body 값을 바로 담을 수 있습니다.

```suggestion
    return ResponseEntity.ok(result);
```

Reply:
반영하겠습니다 ! ㅎㅎ